packages/picasso/src/TableCell/TableCell.tsx

```diff
@@ -49,6 +48,9 @@ export const TableCell = forwardRef<HTMLTableCellElement, Props>(
     const isHead = tableSection === TableSection.HEAD
     const isFooter = tableSection === TableSection.FOOTER
     const titleCase = useTitleCase(propsTitleCase)
+    const contentColor = isFooter ? 'black' : 'dark-grey'
+    const contentSize = isHead ? 'xsmall' : 'small'
+    const contentWeight = isHead || isFooter ? 'semibold' : undefined
```

Comment:
```suggestion
    const fontSettingsBySection = {
      [TableSection.HEAD]: {
        color: 'dark-grey',
        size: 'xsmall',
        weight: 'semibold'
      },
      [TableSection.Footer]: {
        color: 'black',
        size: 'small',
        weight: 'semibold'
      },
      body: {
        color: 'dark-grey',
        size: 'small'
      }
    }
    const { color, size, weight } = fontSettingsBySection[tableSection || 'body]
```

What do you think? I feel little bit lost in all those conditions at first look. This constant can be moved out of component.

Reply:
I like it! 👍  🙇  thanks

Reply:
It's done @TomasSlama feel free to check it out :-)